app/services/pull_request.rb

```diff
@@ -34,7 +34,7 @@ def on_hold_label?(label)
     label.downcase == ON_HOLD.downcase
   end
 
-  def blacklisted?
+  def blocklisted?
     User.where(github_name: username, blacklisted: true).exists?
```

Comment:
In this case, you should create a migration to rename the `blacklisted` attribute in the users table to get consistency